aws/provider.go

```diff
@@ -173,6 +173,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_cloudformation_stack":             dataSourceAwsCloudFormationStack(),
 			"aws_cloudtrail_service_account":       dataSourceAwsCloudTrailServiceAccount(),
 			"aws_cloudwatch_log_group":             dataSourceAwsCloudwatchLogGroup(),
+			"aws_cognito_user_pool_ids":            dataSourceAwsCognitoUserPoolIds(),
```

Comment:
Let's opt to call it `aws_cognito_user_pools` (we may add attributes other than just the ID) - will require refactoring the function names and documentation